openassessment/xblock/static/sass/oa/utilities/_developer.scss

```diff
@@ -570,6 +570,15 @@
     cursor: pointer;
   }
 
+  .openassessment_container_remove_button.is--disabled,
+  .openassessment_container_remove_button.is--disabled:hover,
+  .openassessment_container_add_button.is--disabled,
+  .openassessment_container_add_button.is--disabled:hover {
+    color: $edx-gray-l2;
+    background-color: transparent;
+    cursor: auto;
+  }
+
```

Comment:
There are just two types of buttons in the studio editor; for adding and removing items.